app/helpers/load_balancer_helper/textual_summary.rb

```diff
@@ -28,7 +28,8 @@ def textual_description
 
   def textual_listeners
     @record.load_balancer_listeners.map do |x|
-      "Load Balancer #{x.load_balancer_protocol}:#{x.load_balancer_port}, Instance #{x.instance_protocol}:#{x.instance_port}"
+      "Load Balancer #{x.load_balancer_protocol}:#{x.load_balancer_port_range_str}, Instance"\
```

Comment:
Is protocol always part of the presentation string?  If so, perhaps the `load_balancer_port_range_str` should just do that (this sort of also supports my argument for a general helper method that formats port ranges in a common way).